src/yoroi-wallets/cardano/utxoStorage.ts

```diff
@@ -0,0 +1,136 @@
+import BigNumber from 'bignumber.js'
+
+import storageLegacy from '../../legacy/storage'
+
+// yoroi-lib copy
+export type Asset = {
```

Comment:
are these service types?

Reply:
can we import them from the lib, maybe they should be exposed as ServiceTypes. ... 🤔

Reply:
yeah, do not do not do not copy types from another library

Reply:
Unfortunately those types aren't exposed

Reply:
I've opened a PR to exposed them.